scripts/run.sh

```diff
@@ -85,14 +85,14 @@ function install() {
     
             mkdir -p $OUTPUT_DIR/letsencrypt
             docker pull certbot/certbot
-            docker run -it --rm --name certbot -p 80:80 -v $OUTPUT_DIR/letsencrypt:/etc/letsencrypt/ certbot/certbot \
+            docker run --rm --name certbot -p 80:80 -v $OUTPUT_DIR/letsencrypt:/etc/letsencrypt/ certbot/certbot \
                 certonly --standalone --noninteractive  --agree-tos --preferred-challenges http \
                 --email $EMAIL -d $DOMAIN --logs-dir /etc/letsencrypt/logs
         fi
     fi
     
     pullSetup
-    docker run -it --rm --name setup -v $OUTPUT_DIR:/bitwarden \
+    docker run -i --rm --name setup -v $OUTPUT_DIR:/bitwarden \
```

Comment:
Is this going to break interaction installs though?

Reply:
No, in interactive mode the setup container only needs to read the id and key from stdin, it does not require the pseudo terminal.